mojaloop/values.yaml

```diff
@@ -22,3187 +28,3808 @@ CONFIG:
   obj_mongo_host: &OBJSTORE_MONGO_HOST 'cl-mongodb'
   obj_mongo_port: &OBJSTORE_MONGO_PORT 27017
   obj_mongo_user: &OBJSTORE_MONGO_USER mojaloop
-  obj_mongo_password: &OBJSTORE_MONGO_PASSWORD password
+  obj_mongo_password: &OBJSTORE_MONGO_PASSWORD ''
+  obj_mongo_secret: &OBJSTORE_MONGO_SECRET
+    name: &OBJSTORE_MONGO_SECRET_NAME cl-mongodb
+    key: &OBJSTORE_MONGO_SECRET_KEY mongodb-passwords
   obj_mongo_database: &OBJSTORE_MONGO_DATABASE mlos
-  objstore_uri: &OBJSTORE_URI 'mongodb://mojaloop:password@cl-mongodb:27017/mlos'
 
-  ## CENTRAL-EVEN-PROCESSOR Backend
+  ## TTK MONGODB BACKEND
+  ttk_mongo_host: &TTK_MONGO_HOST "ttk-mongodb"
+  ttk_mongo_port: &TTK_MONGO_PORT "27017"
+  ttk_mongo_user: &TTK_MONGO_USER "ttk"
+  ttk_mongo_password: &TTK_MONGO_PASSWORD ""
+  ttk_mongo_secret: &TTK_MONGO_SECRET
+    name: &TTK_MONGO_SECRET_NAME ttk-mongodb
+    key: &TTK_MONGO_SECRET_KEY mongodb-passwords
+  ttk_mongo_database: &TTK_MONGO_DATABASE "ttk"
+
+  ## CENTRAL-EVEN-PROCESSOR BACKEND
   cep_mongo_host: &CEP_MONGO_HOST 'cep-mongodb'
   cep_mongo_port: &CEP_MONGO_PORT 27017
   cep_mongo_user: &CEP_MONGO_USER mojaloop
   cep_mongo_password: &CEP_MONGO_PASSWORD password
```

Comment:
```suggestion
  cep_mongo_password: &CEP_MONGO_PASSWORD ''
```

Reply:
@elnyry-sam-k this is arelady set on the top-level config.